From 58334377da5bd1c106b658ae6ca89907264372bb Mon Sep 17 00:00:00 2001
From: Andreas Mohr <and@gmx.li>
Date: Sat, 20 Feb 2016 16:12:31 +0000
Subject: [PATCH] setup.c: cleanup -Wcast-qual warning

setup.c: In function 'load_setup':
setup.c:1121:14: warning: cast discards 'const' qualifier from pointer target type [-Wcast-qual]
     buffer = (char *) get_codepage_id (mc_global.display_codepage);
              ^

Signed-off-by: Andreas Mohr <and@gmx.li>
---
 src/setup.c | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/src/setup.c b/src/setup.c
index adb021d..cf0b0f9 100644
--- a/src/setup.c
+++ b/src/setup.c
@@ -998,6 +998,7 @@ load_setup (void)
 
 #ifdef HAVE_CHARSET
     char *buffer;
+    const char *cbuffer;
 
     load_codepages_list ();
 #endif /* HAVE_CHARSET */
@@ -1118,9 +1119,9 @@ load_setup (void)
         is_autodetect_codeset_enabled = TRUE;
 
     g_free (init_translation_table (mc_global.source_codepage, mc_global.display_codepage));
-    buffer = (char *) get_codepage_id (mc_global.display_codepage);
-    if (buffer != NULL)
-        mc_global.utf8_display = str_isutf8 (buffer);
+    cbuffer = get_codepage_id (mc_global.display_codepage);
+    if (cbuffer != NULL)
+        mc_global.utf8_display = str_isutf8 (cbuffer);
 #endif /* HAVE_CHARSET */
 
 #ifdef HAVE_ASPELL
